srcpkgs/airspyhf/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'airspyhf'
+pkgname=airspyhf
+version=1.6.8
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libusb-devel"
+short_desc="Usermode driver and tools for Airspy HF+"
+maintainer="classabbyamp <void@placeviolette.net>"
+license="BSD-3-Clause"
+homepage="https://github.com/airspy/airspyhf"
+distfiles="https://github.com/airspy/airspyhf/archive/refs/tags/${version}.tar.gz"
+checksum=cd1e5ae89e09b813b096ae4a328e352c9432a582e03fd7da86760ba60efa77ab
+
+post_install() {
+	vlicense LICENSE
+}
+
+libairspyhf_package() {
+	short_desc+=" - library"
+	pkg_install() {
+		vmove usr/lib/*.so.*
+		vsed -e 's/GROUP="plugdev"/\0, TAG+="uaccess"/' -i tools/52-airspyhf.rules
+		vinstall tools/52-airspyhf.rules 644 usr/lib/udev/rules.d/
+	}
+}
+
+libairspyhf-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
```

Comment:
The pkgconfig file mentions libusb in Cflags, but not in Requires, do you know if that's a bug upstream?
```
Cflags: -I${includedir}/libairspyhf/ -I/usr/include/libusb-1.0
```

Reply:
doesn't seem to be reported anywhere. The `.pc.in` file doesn't have a requires line either.

```
$ xbps-query -R --cat /usr/lib/pkgconfig/libairspy.pc libairspy-devel
prefix=/usr
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: AirSpy Library
Description: C Utility Library
Version: 1.0
Cflags: -I${includedir}/libairspy/ -I/usr/include/libusb-1.0
Libs: -L${libdir} -lairspy
Libs.private: 
```
the other library from airspy has the same thing pretty much

Reply:
I checked repology to see if any other distros had a patch for this, but I didn't find anything. I did find one useful patch though, which I've added.

Will report the libusb thing upstream